src/org/rascalmpl/parser/gtd/stack/LiteralStackNode.java

```diff
@@ -15,92 +15,113 @@
 import org.rascalmpl.parser.gtd.result.LiteralNode;
 import org.rascalmpl.parser.gtd.stack.filter.ICompletionFilter;
 import org.rascalmpl.parser.gtd.stack.filter.IEnterFilter;
+import org.rascalmpl.unicode.UnicodeConverter;
 
 public final class LiteralStackNode<P> extends AbstractMatchableStackNode<P>{
 	private final int[] literal;
 	private final P production;
-	
+
 	private final LiteralNode result;
-	
+
 	public LiteralStackNode(int id, int dot, P production, int[] literal){
 		super(id, dot);
-		
+
 		this.literal = literal;
 		this.production = production;
-		
+
 		result = new LiteralNode(production, literal);
 	}
-	
+
 	public LiteralStackNode(int id, int dot, P production, int[] literal, IEnterFilter[] enterFilters, ICompletionFilter[] completionFilters){
 		super(id, dot, enterFilters, completionFilters);
-		
+
 		this.literal = literal;
 		this.production = production;
-		
+
 		result = new LiteralNode(production, literal);
 	}
-	
+
+	public int[] getLiteral() {
+		return literal;
```

Comment:
This is a judgement call, but I think it might have an impact on performance and I am very sure I do not change the content of `literal`. It is a pity Java does not have the concept of `const` like in C++.